lua/refactoring/treesitter/treesitter.lua

```diff
@@ -65,11 +65,19 @@ function TreeSitter:loop_thru_nodes(scope, inline_nodes)
     return out
 end
 
-function TreeSitter:get_local_var_names(node)
+function TreeSitter:get_all_local_var_names(node)
+    return self:loop_thru_nodes(node, self.local_var_names)
+end
+
+function TreeSitter:get_first_local_var_name(node)
     return self:loop_thru_nodes(node, self.local_var_names)[1]
 end
 
-function TreeSitter:get_local_var_values(node)
+function TreeSitter:get_all_local_var_values(node)
```

Comment:
Do we just want to change this to get local vars and then set it to to get the first index where needed? Don't want to repeat same logic minus the index.

Reply:
Yeah that makes sense, I'll change that.